### PR TITLE
everything is aligned

### PR DESCRIPTION
**Pull request by mmalonga**
self explanatory

### CODE REVIEW

File: app/src/main/java/com/github/se/gatherspot/ui/eventUI/EventDataForm.kt [hunk 1]
```diff
@@ -84,6 +84,7 @@ import kotlinx.coroutines.runBlocking
 
 private val WIDTH = 300.dp
 private val WIDTH_2ELEM = 100.dp
+private val WIDTHWITHPICKER = WIDTH.times(0.8f)
```
Comment (SamBachmann) [inline app/src/main/java/com/github/se/gatherspot/ui/eventUI/EventDataForm.kt:87]: To improve readability, it is better to use SNAKE_CASE (with underscore between words)

**Review by SamBachmann — APPROVED** [review]
Good refactoring, please juste change the constant name indicated above

File: app/src/main/java/com/github/se/gatherspot/ui/eventUI/EventDataForm.kt (hunk 1 above)
Comment (mmalonga) [reply app/src/main/java/com/github/se/gatherspot/ui/eventUI/EventDataForm.kt:87]: Yes but Android Studio sends a warning saying its not common practice for files that aren't top level

Comment (SamBachmann) [reply app/src/main/java/com/github/se/gatherspot/ui/eventUI/EventDataForm.kt]: okee

**Pull request merged**